.goreleaser

```diff
@@ -0,0 +1,68 @@
+project_name: root-signing
+
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+  - COSIGN_EXPERIMENTAL=true
+
+# Prevents parallel builds from stepping on eachothers toes downloading modules
+before:
+  hooks:
+    - go mod tidy
+    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+
+gomod:
+  proxy: true
+
+sboms:
+  - artifacts: binary
+
+builds:
+  - id: root-verify
+    binary: root-verify-linux-{{ .Arch }}
+    no_unique_dist_dir: true
+    main: ./cmd/verify
+    goos:
+      - linux
+    goarch:
+      - amd64
```

Comment:
For now, just linux/amd64 since we only need this for probers. I think if more people require it (like maybe during the ceremonies) we can expand to other releases soon too